src/components/Display.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ total }) => <div className="display">{total || '0'}</div>;
```

Comment:
- [ ] Although this works, per the requirement of this milestone, you are supposed to set default props in the following manner. Just add something similar to this before exporting.
```
Display.defaultProps = {
  someProp: 'Some Value',
};
```